hello-world-python/enclave/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+    id 'com.r3.conclave.enclave'
+}
+
+dependencies {
+    testImplementation "org.junit.jupiter:junit-jupiter:5.6.0"
+}
+
+conclave {
+    productID = 1
+    revocationLevel = 0
+
+    simulation {
+        signingType = privateKey
+        signingKey = file("../signing/sample_private_key.pem")
+        supportLanguages = "python"
```

Comment:
Shouldn't 'supportLanguages = "python"' below revocationLevel? That is how the docs do.

Reply:
Yes correct. Let me then try to run one more time in simulation mode now then.

Reply:
updated the position. Thanks for pointing this out.

Reply:
This change has not been addressed.

Reply:
done. apologies this was committed but not pushed.